src/constants.ts

```diff
@@ -99,7 +101,9 @@ export type Station = {
   lastLocalCheckTime: Date
   clickTimestamp: Date
   clickCount: number
-  clickTrend: number
+  clickTrend: number,
+  geo_lat?: number
+  geo_long?: number
```

Comment:
New properties should use camel case like the rest,
```suggestion
  geoLat?: number
  geoLong?: number
```